lib/crack.rb

```diff
@@ -0,0 +1,13 @@
+require './lib/enigma.rb'
+
+enigma = Enigma.new
+
+message_file = File.open(ARGV[0],"r")
+
+encrypt_message = enigma.crack(message_file.read,ARGV[2])
```

Comment:
Ah, after reading it a few times it looks like a typo: this should be "cracked_message".